Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_BezierPointer.cs

```diff
@@ -28,6 +28,9 @@ public class VRTK_BezierPointer : VRTK_WorldPointer
         public GameObject customPointerCursor;
         public LayerMask layersToIgnore = Physics.IgnoreRaycastLayer;
 
+        [Tooltip("Optional object that appears\nwhen the teleport is allowed")]
+        public GameObject teleportBeam = null;
```

Comment:
I'm not sure I like the name `Teleport Beam` - Is it specific to teleporting or is it just when a destination is valid?

If it's to do with teleporting only then maybe something more descriptive like `Valid Teleport Location Object`